packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -992,6 +992,37 @@ export class CompletionList {
     }
 }
 
+export enum InlineCompletionTriggerKind {
+    Invoke = 0,
+    Automatic = 1,
+}
+
+@es5ClassCompat
+export class InlineSuggestion implements theia.InlineCompletionItem {
```

Comment:
Question: Is there a specific reason to deviate from the original `InlineCompletionItem` name for this class implementation? Same question for the `InlineSuggestionList` below.

Reply:
@msujew I agree, I wasn't too fond of having the different names here as well but it came straight from VS Code. I don't think we need to follow in this case.

Reply:
I see, interesting. Yeah, we can probably get away with changing the class name to `InlineCompletionItem`.